broken-site-reporting/README.md

```diff
@@ -25,6 +25,8 @@ Test suite specific fields:
 - `model` - string - name of the device model (native apps only)
 - `os` - string - operating system name (native apps only)
 - `gpcEnabled` - boolean - if GPC is enabled or not (native apps only) - GPC can be disabled by user or by remote config
+- `errorDescription` - string - optional field containing a description from an error generated by failed page load
```

Comment:
Like below, so name as `errorDescriptions`

Test case: `["net::ERR_NAME_NOT_RESOLVED","net::ERR_ABORTED"]` should become `%5B%22net%3A%3AERR_NAME_NOT_RESOLVED%22%2C%22net%3A%3AERR_ABORTED%22%5D`